blockdag/dag.go

```diff
@@ -1366,6 +1366,23 @@ func (dag *BlockDAG) BlockConfirmationsByHashNoLock(hash *daghash.Hash) (uint64,
 	return dag.blockConfirmations(node)
 }
 
+// TxConfirmations returns the confirmations for the given outpoint, if it exists
+// in the DAG.
+//
+// This function is safe for concurrent access.
+func (dag *BlockDAG) TxConfirmations(outpoint *wire.Outpoint) (uint64, bool) {
```

Comment:
Rename to UTXO confirmations or something, since it's never dealing with whould Txs.